### DISCUSSION TITLE
Backward referencies

### DISCUSSION THREAD

**Discussion by jlp656** [duckdb/duckdb#12475] [Q&A]
Bonjour. L'implémentation des backward referencies est-elle prévue à terme dans les requêtes avec des expressions rationnelles dans Duckdb ? Merci

**Comment by Tishj** [comment]
Since we rely on RE2 for regexp execution I don't think that functionality is coming, judging from their wikipedia page: <https://en.wikipedia.org/wiki/RE2_(software)>

I could be wrong, you would have to check their github repo

**Reply by jlp656** [reply]
OK. Thank you for your reply !

**Reply by soerenwolfers** [reply]
GitHub confirms: https://github.com/google/re2/issues/101
